packages/mattermost-redux/src/actions/search.ts

```diff
@@ -299,6 +299,31 @@ export function removeSearchTerms(teamId: string, terms: string): ActionFunc {
     };
 }
 
+export function getRecentSearches(): ActionFunc {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        dispatch({type: SearchTypes.RECENT_SEARCHES_REQUEST});
```

Comment:
Unless you're specifically using them for something here, could you remove the `RECENT_SEARCHES_REQUEST` and `RECENT_SEARCHES_FAILURE` actions dispatched here? They're part of an old pattern where we used to track request state, so we don't want to add them for new code

Reply:
I don't use them, I will remove them then :+1: